ddtrace/contrib/internal/openai/patch.py

```diff
@@ -162,6 +180,23 @@ def patched_client_init(openai, pin, func, instance, args, kwargs):
     return
 
 
+@with_traced_module
+def patched_completions_with_raw_response_init(openai, pin, func, instance, args, kwargs):
```

Comment:
Let's add a docstring here to note what/why we're doing this here for future readers.